modules/react-native-status/ios/RCTStatus/RCTStatus.m

```diff
@@ -318,6 +318,61 @@ - (void)handleSignal:(NSString *)signal
     callback(@[result]);
 }
 
+//////////////////////////////////////////////////////////////////// getConnectionStringForBootstrappingAnotherDevice
+RCT_EXPORT_METHOD(getConnectionStringForBootstrappingAnotherDevice:(NSString *)configJSON
+                  callback:(RCTResponseSenderBlock)callback) {
+
+    NSData *configData = [configJSON dataUsingEncoding:NSUTF8StringEncoding];
+    NSDictionary *configDict = [NSJSONSerialization JSONObjectWithData:configData options:NSJSONReadingMutableContainers error:nil];
+    NSLog(@"configDict is ====> %@", configDict);
+    NSString *keyUID = [configDict objectForKey:@"keyUID"];
+    NSLog(@"keyUID is ====> %@", keyUID);
+//    NSString *keystoreDir = [@"/keystore/" stringByAppendingString:keyUID];
+   NSURL *multiaccountKeystoreDir = [self getKeyStoreDir:keyUID];
+//     NSFileManager *fileManager = [NSFileManager defaultManager];
+//     NSURL *rootUrl =[[fileManager URLsForDirectory:NSLibraryDirectory inDomains:NSUserDomainMask] lastObject];
+//     NSURL *multiaccountKeystoreDir = [rootUrl URLByAppendingPathComponent:@"keystore"];
+    NSString *keystoreDir = multiaccountKeystoreDir.path;
+
+    [configDict setValue:keystoreDir forKey:@"keystorePath"];
+    NSString *modifiedConfigJSON = [configDict bv_jsonStringWithPrettyPrint:NO];
+    NSLog(@"modifiedConfigJSON is ====> %@", modifiedConfigJSON);
+
+#if DEBUG
```

Comment:
do we want/need to maintain all of these DEBUG cases?

Reply:
for the time being yes, it helps , I will get rid of this in follow up PRs